src/gui/widgets/RenameDialog.cpp

```diff
@@ -56,6 +57,13 @@ RenameDialog::~RenameDialog()
 
 
 
+void RenameDialog::resizeEvent (QResizeEvent * event) {
+	m_stringLE->setGeometry ( 10, 5, width() - 20, 20 );	
```

Comment:
Side note on this... I feel we shouldn't get too comfortable with the resize hacks, but rather use containers which can handle resizing properly.  That said, looks god.